.vscode/cspell.json

```diff
@@ -174,7 +174,8 @@
     "westcentralus",
     "westus",
     "Wmem",
-    "xunit"
+    "xunit",
+    "Rdma"
```

Comment:
Please insert this into the list in alphabetical order

Reply:
ok